src/components/ReportingConfig/ReportingConfigForm.jsx

```diff
@@ -404,7 +406,13 @@ ReportingConfigForm.propTypes = {
     uuid: PropTypes.string,
     title: PropTypes.string,
   })).isRequired,
-  reportingConfigTypes: PropTypes.arrayOf(PropTypes.arrayOf(PropTypes.string)).isRequired,
+  reportingConfigTypes: PropTypes.shape({
+    dataType: PropTypes.array,
```

Comment:
is there an arrayOf we can use here to be more exact on what type of array?

Reply:
yeah I would rather the type system (TS or language) handle typing but in absence of it, we get to do this fun type stuff lol)